toc/rfc/rfc-draft-efficient-modern-vm-sizes.md

```diff
@@ -0,0 +1,118 @@
+# Meta
+[meta]: #meta
+- Name: CF and BOSH should default to more efficient and smaller performance VM sizes
+- Start Date: 2022-05-30
+- Author(s): @dsboulder
+- Status: Draft
+- RFC Pull Request: [#290](https://github.com/cloudfoundry/community/pull/290)
+
+
+## Summary
+
+The vast majority of CF and BOSH deployments are deployed automatically in CF
+testing pipelines (or in proprietary vendor pipelines), so we should optimize
+the default VM sizes for these workloads. We will choose VM sizes that are
+either cheaper because they are newer generation, they provide a smaller/finer
+tuned amount of hardware, or they have burstable cores for appropriate
+workloads.
+
+**Note: This is a breaking change, especially for [Reserved
+Instance](https://aws.amazon.com/ec2/pricing/reserved-instances/) holders.
+Anyone using the default VM sizes for a high-performance or production CF should
+consider customizing their VM sizes. In addition, the Diego cells have been
+reduced from 26-64GB of memory to a consistent 16GB by default, so please
+consider how many you need or resize them up.**.
+
+We believe it is acceptable to make this type of change because we expect
+production deployments of CF to customize their VM types anyway, and we cannot
+prevent progress just to help RI buyers.
+
+## Problem
+
+We'd like to save the CFF, and anyone who deploys default CF configuration,
+significant money in operating BOSH and CF deployment. The current deployment
+costs for CF-d are so expensive (for example on AWS) that our limited foundation
+CI budget will be consumed too quickly. In addition, our VM type usages are in
+need of a refresh to match the newest IaaS offerings for both performance and
+cost.
+
+## Proposal
```

Comment:
Should we be doing anything going forward to prevent us falling behind on IaaS offerings again?

Reply:
Psh, we should do this exercise again at some pt? But I don't need to write that down